dev_guide/builds.adoc

```diff
@@ -1557,3 +1557,84 @@ Docker and Source builds set the following labels on output images:
 |*io.openshift.build.source-location*
 |Source URL for the build
 |===
+
+
+[[using-external-artifacts]]
+= Using External Artifacts During The Build
+
+In order to use external artifacts during the build you don't need any special
+support from OpenShift platform. You could download and use any artifacts
+as you want.
+
+For `*Source*` build strategy you need to put appropriate shell commands into
+the `assemble` script.
+
+.`.sti/bin/assemble` file
+====
+
+[source,bash]
+----
+#!/bin/sh -x
+wget http://code.jquery.com/jquery-2.1.4.min.js
+----
```

Comment:
This assemble script is not really usable the way it is, it would override the default assemble script and not really "build".
I'd try to make it a more realistic example.